clang/test/CodeGenCXX/attr-cpuspecific-outoflinedefs.cpp

```diff
@@ -80,17 +80,17 @@ OutOfLineDefs::foo(int, int, int) {
 // LINUX: define dso_local noundef i32 @_ZN13OutOfLineDefs3fooEiii.S
 // LINUX: define dso_local noundef i32 @_ZN13OutOfLineDefs3fooEiii.R
 // LINUX: define weak_odr ptr @_ZN13OutOfLineDefs3fooEiii.resolver()
-// LINUX: ret ptr @_ZN13OutOfLineDefs3fooEiii.R
 // LINUX: ret ptr @_ZN13OutOfLineDefs3fooEiii.S
+// LINUX: ret ptr @_ZN13OutOfLineDefs3fooEiii.R
```

Comment:
Is it affected by `f16c` feature? I'm worried if the test will fail with some old GCC version.

Reply:
Yes, I guess it will. Need some environment to verify.

Reply:
We landed https://reviews.llvm.org/D158811 before, I think this won't be blocking issue? 
CC @MaskRay

Reply:
Reordering `X86_FEATURE` causes the change here. This should not matter. Ideally, the test can be improved to be check the relevant `__cpu_model` members.

Reply:
Thanks comment!